sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py

```diff
@@ -121,6 +121,7 @@ def _get_partitions(self):
             for p_id in cast(List[str], self._partition_ids):
                 self._producers[p_id] = None
 
+    # this is spelled wrong here -- do we change - or were the changes elsewhere
     def _get_max_mesage_size(self):
```

Comment:
I know we changed this elsewhere -- will also change here doesn't seem to have been integrated yet?